### PR TITLE
Monitoring in Dev Console workflow

### PR DESCRIPTION
**Pull request by Preeticp**
This PR is targeted for the 4.4 release
This has been reviewed by SMEs and QE

### CODE REVIEW

File: modules/odc-monitoring-your-project-metrics.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+= Monitoring your project metrics
+
+After you create applications in your project and deploy them, you can use the *Developer* perspective in the web console to see the metrics for your project.
+On the left navigation panel of the *Developer* perspective, click *Monitoring* to see the *Dashboard*, *Metrics*, and *Events* for your project.
+
+.Procedure
+
+* Use the *Dashboard* tab to see graphs depicting the CPU, memory, and bandwidth consumption and network related information, such as the rate of transmitted and received packets and the rate of dropped packets.
++
+image::odc_project_dashboard.png[Monitoring dashboard]
```
Comment (adellape) [inline modules/odc-monitoring-your-project-metrics.adoc]: For each of the screenshots, consider adding an image title:

```
.Image title
image::file.png[Alt text]
```

File: modules/odc-monitoring-your-project-metrics.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
++
+Use the following options to see further details:
+
+** Select the *Time Range* from the drop-down to determine the time frame for the data being captured.
```
Comment (adellape) [inline modules/odc-monitoring-your-project-metrics.adoc]: I believe per IBM Style Guide, each instance of "drop-down" throughout should be more specifically "drop-down list"?

File: modules/odc-monitoring-your-project-metrics.adoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc
+
+[id="odc-monitoring-your-project-metrics_{context}"]
+= Monitoring your project metrics
+
+After you create applications in your project and deploy them, you can use the *Developer* perspective in the web console to see the metrics for your project.
+On the left navigation panel of the *Developer* perspective, click *Monitoring* to see the *Dashboard*, *Metrics*, and *Events* for your project.
```
Comment (adellape) [inline modules/odc-monitoring-your-project-metrics.adoc]: Could this sentence with its "click" instruction instead be the first step of the Procedure possibly?

File: modules/odc-monitoring-your-project-metrics.adoc [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+* Use the *Dashboard* tab to see graphs depicting the CPU, memory, and bandwidth consumption and network related information, such as the rate of transmitted and received packets and the rate of dropped packets.
++
+image::odc_project_dashboard.png[Monitoring dashboard]
++
+Use the following options to see further details:
+
+** Select the *Time Range* from the drop-down to determine the time frame for the data being captured.
+** Select the *Refresh Interval* from the drop-down to determine the time period after which the data is refreshed.
+** Hover your cursor over the graphs to see specific details for your pod.
```
Comment (adellape) [inline modules/odc-monitoring-your-project-metrics.adoc]: Throughout:

s/pod/Pod/

File: modules/odc-monitoring-your-project-metrics.adoc [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+[id="odc-monitoring-your-project-metrics_{context}"]
+= Monitoring your project metrics
+
+After you create applications in your project and deploy them, you can use the *Developer* perspective in the web console to see the metrics for your project.
+On the left navigation panel of the *Developer* perspective, click *Monitoring* to see the *Dashboard*, *Metrics*, and *Events* for your project.
+
+.Procedure
+
+* Use the *Dashboard* tab to see graphs depicting the CPU, memory, and bandwidth consumption and network related information, such as the rate of transmitted and received packets and the rate of dropped packets.
```
Comment (adellape) [inline modules/odc-monitoring-your-project-metrics.adoc]: Are these steps purposefully unordered?

File: modules/odc-monitoring-your-application-metrics.adoc [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+.Procedure
+
+. In the *Topology* view, click the application to see the application details in the right panel.
+. Click the *Monitoring* tab to see the warning events for the application; graphs for CPU, memory, and bandwidth usage; and all the events for the application.
++
+image::odc_app_metrics.png[Monitoring application metrics]
++
+.. You can click any of the charts to go to the *Metrics* tab to see the detailed metrics for the application.
+.. You can click *View monitoring dashboard* to see the monitoring dashboard for that application.
```
Comment (adellape) [inline modules/odc-monitoring-your-application-metrics.adoc]: Consider dropping both of the "You can"s in each of these substeps?

**Review by adellape — COMMENTED** [review]
Looks good! Just minor suggestions inline.

File: modules/odc-monitoring-your-project-metrics.adoc (hunk 5 above)
Comment (Preeticp) [reply modules/odc-monitoring-your-project-metrics.adoc]: Yes, technically an user can go either to the Dashboard, Metrics, or Events tab. They do not need to be done in this order.

File: modules/odc-monitoring-your-project-metrics.adoc (hunk 2 above)
Comment (Preeticp) [reply modules/odc-monitoring-your-project-metrics.adoc]: Using this instruction from the IBM style guide: "Refer to a drop-down list by its label, followed by list.' (Chpt 7- Computer interfaces, pg 209) I changed the mentions of drop-down to list in all cases that had a label. There is this one case which does not have a label on L-32 there I have changed it to drop-down list.

File: modules/odc-monitoring-your-project-metrics.adoc (hunk 1 above)
Comment (Preeticp) [reply modules/odc-monitoring-your-project-metrics.adoc]: That's a good suggestion, I will need to do this for all the screenshots in the ODC docs to keep it consistent. If you don't mind I will do that as a separate PR. I have created a Jira issue here: https://issues.redhat.com/browse/RHDEVDOCS-1736

**Comment by Preeticp** [comment]
@adellape I have fixed most of your suggestions except for the images title for which I have created an issue and will tackle all the ODC docs images at one go. Please let me know if anything else needs to be modified. Request you to merge if you find the changes alright.

**Comment by adellape** [comment]
@Preeticp Thanks! Done.

**Comment by Preeticp** [comment]
Thank you @adellape

**Pull request merged**